jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp

```diff
@@ -32,7 +32,7 @@
 
 <div class="container form-login-related registration-page">
   <form:form id="form" name="form" action='${pageContext.request.contextPath}/user/new'
-             modelAttribute="newUser" method="POST" class='form-vertical'>
+             modelAttribute="newUser" method="POST" class='form-vertical' tabindex="-1">
```

Comment:
Try on QA, PREPROD has an old version. Make sure you're trying the page, not the dialog.